README.md

```diff
@@ -4,3 +4,121 @@ A central place for introspecting and executing the various modular components o
 [![Tests](https://github.com/pangeo-forge/pangeo-forge-orchestrator/actions/workflows/main.yaml/badge.svg)](https://github.com/pangeo-forge/pangeo-forge-orchestrator/actions/workflows/main.yaml)
 [![pre-commit](https://github.com/pangeo-forge/pangeo-forge-orchestrator/actions/workflows/pre-commit.yaml/badge.svg)](https://github.com/pangeo-forge/pangeo-forge-orchestrator/actions/workflows/pre-commit.yaml)
 [![Documentation Status](https://readthedocs.org/projects/pangeo-forge-orchestrator/badge/?version=latest)](https://pangeo-forge-orchestrator.readthedocs.io/en/latest/?badge=latest)
+
+
+## Database Migrations with Alembic
+
+This application is configured to use Alembic database migrations.
+The configuration was pieced together from the following resources:
+
+- Generic Alembic tutorial: https://alembic.sqlalchemy.org/en/latest/tutorial.html
+- Using sqlmodel with Alembic: https://github.com/tiangolo/sqlmodel/issues/85#issuecomment-917228849
+
+### Relevant Files
+
+- `alembic.ini` - Alembic configuration file
+- `migrations/env.py` - The migration environment; customized to work with
+  sqlmodel our test / deployment environment
+- `migrations/versions/*.py` - Auto-generated scripts which perform the migrations.
+
+### `DATABASE_URL` Environment Variable
+
+Both the migration environment and the FastAPI app itself are configured to use
+the `DATABASE_URL` environment variable to determine which database to connect to.
+**This environment variable must be set in order for the app to work.**
+This string must be a valid sqlalchemy database URI.
+
+For example, to set up a local environment for the test suite:
+
+```bash
+export DATABASE_URL=sqlite:///`pwd`/database.sqlite
+pytest -vx tests
+```
+
```

Comment:
```suggestion
```

This doesn't work without running migrations, which are not described until the following section. I propose moving this content into its own heading below.